yarn.lock

```diff
@@ -13378,11 +13378,16 @@ prettier-linter-helpers@^1.0.0:
   dependencies:
     fast-diff "^1.1.2"
 
-prettier@^2.0.4, prettier@^2.0.5:
+prettier@^2.0.5:
```

Comment:
#2097 is merged but I still see yarn.lock changes here. May need to merge `master` into this branch?